modular_skyrat/modules/customization/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -17,5 +17,6 @@
 	var/oversized_piggywound_chance = 50
 	/// Base damage for oversized piggyback riding.
 	var/oversized_piggydam = 25
-	/// Knockdown time for oversized piggyback riding in deciseconds. (10 deciseconds = 1 second)
-	var/oversized_piggyknock = 30
+	/// Paralyze time for oversized piggyback riding in deciseconds. (10 deciseconds = 1 second)
+	var/oversized_piggyparalyze = 30
```

Comment:
use SECONDS defines

Reply:
> use SECONDS defines

So, should I use seconds in defines and just multiply it by ten in the code?

Reply:
No, there is a `SECONDS` define for times. Use it like `= 3 SECONDS` and the result will be 30.